task-03/get_top_names.py

```diff
@@ -4,16 +4,34 @@
 https://github.com/astropgh/learning-by-doing/tree/master/task-03
 """
 
-def extract_data_lines(filename, start_text, end_text):
+def extract_data_lines(
+    filename,
+    start_text,
+    end_text,
+    include_start=False,
+    include_end=False,
+):
     """
     open `filename`, and yield the lines between
     the line that contains `start_text` and the line that contains `end_text`
     """
     # fill in code as needed
     with open(filename) as fh:
+        in_table = False
         for line in fh:
-            # fill in code as needed
-            # use `yield line` to return desired lines but keep the function going
+
+            if line.strip() == start_text:
```

Comment:
The docstring says line that *contains* `start_text`, so here it would be better to do:
```python
if start_text in line:
```